cohere/client.py

```diff
@@ -455,7 +462,7 @@ def __request(self, endpoint, json=None) -> Any:
 
         url = urljoin(self.api_url, endpoint)
         if use_xhr_client:
-            response = self.__pyfetch(url, headers, json.dumps(json))
+            response = self.__pyfetch(url, headers, jsonlib.dumps(json))
```

Comment:
`json` argument shadows `json` library, resulting in `json.dumps` being called on request body